source/client-side-operations-timeout/tests/convenient-transactions.yml

```diff
@@ -103,3 +100,11 @@ tests:
               command:
                 insert: *collectionName
                 maxTimeMS: { $$type: ["int", "long"] }
+          - commandStartedEvent:
```

Comment:
As `withTransaction` expects an error, then an `abortTransaction` event will be issued.